R/lint.R

```diff
@@ -111,6 +91,33 @@ lint <- function(filename, linters = NULL, ..., cache = FALSE, parse_settings =
   zap_temp_filename(res, needs_tempfile)
 }
 
+lint_impl_ <- function(linters, lint_cache, filename, source_expressions) {
```

Comment:
we could revert this -- it's only done to keep the cyclomatic complexity down, which had ballooned a bit at the earlier changes, now simplified again.

otoh, it does seem like a natural place for a helper; leaving it in for now.